src/components/StudioTopbar/StudioTopbar.style.tsx

```diff
@@ -0,0 +1,186 @@
+import React from 'react'
+import styled from '@emotion/styled'
+import { Link } from 'react-router-dom'
+import { breakpoints, colors, sizes, transitions, typography } from '@/shared/theme'
+import { TopbarBase, Avatar, Text, Button, TOP_NAVBAR_HEIGHT, TopbarBaseProps } from '@/shared/components'
+
+type ChannelInfoProps = {
+  active?: boolean
+}
+
+type NavDrawerProps = {
+  active?: boolean
+}
+
+type DrawerButtonProps = {
+  isActive?: boolean
+}
+
+export const StyledTopbarBase = styled((props: TopbarBaseProps) => <TopbarBase {...props} />)`
+  display: flex;
+  justify-content: space-between;
+  @media screen and (min-width: ${breakpoints.small}) {
+    display: flex;
+    justify-content: space-between;
+  }
+`
+
+export const DrawerButton = styled(Button)<DrawerButtonProps>`
+  svg {
+    transform: rotate(${({ isActive }) => (isActive ? '180deg' : '0')});
+    transition: transform ${transitions.timings.regular} ${transitions.easing};
+  }
+`
+
+export const ChannelInfoContainer = styled.div<ChannelInfoProps>`
+  display: flex;
+  align-items: center;
+  background-color: ${({ active }) => active && colors.gray[600]};
+  &:hover {
+    cursor: pointer;
+    background-color: ${({ active }) => (active ? colors.gray[600] : colors.gray[700])};
+  }
+  svg {
+    margin-left: auto;
+    margin-right: 12px;
+  }
+`
+
+export const StyledAvatar = styled(Avatar)`
+  width: 42px;
+  height: 42px;
```

Comment:
It's not in the scope of this task, but maybe we could add these sizes as a variant to the Avatar component? 
Just as a sidenote.